cmd/draft/connect.go

```diff
@@ -102,6 +103,10 @@ func (cn *connectCmd) run(runningEnvironment string) (err error) {
 		}
 	}
 
+	go func() {
```

Comment:
Can we simplify this down to just `go runTasks(tasks.Setup, connection.PodName)` or am I missing something?